Event_README.md

```diff
@@ -0,0 +1,67 @@
+## Introduction to Excel 1.8 Event Features
+
+This page describes the Excel Event APIs that are planned for the next release (requirement set 1.8). 
+
+_**Note**: The features listed are still in the design and review phase and are not yet available as part of the product. The final design is subject to change. When the feature is made available, the final specification will be published as part of the master repository._
+
+_Details of the specific APIs are listed below. Please let us know what you think!_
```

Comment:
Please change the sentence here to say "The following table provides details about the specific APIs."